composer.json

```diff
@@ -98,7 +98,8 @@
         "symfony/validator": "^4.3",
         "symfony/yaml": "^4.3",
         "toflar/psr6-symfony-http-cache-store": "^1.1 || ^2.0",
-        "twig/twig": "^1.41 || ^2.0"
+        "twig/twig": "^1.41 || ^2.0",
+        "symfony/string": "^5.0"
```

Comment:
You've added this manually right? If you add it using composer it would be alphabetically sorted :slightly_smiling_face: Can you please change that?